manager/state/store/memory.go

```diff
@@ -185,6 +211,7 @@ type readTx struct {
 
 // View executes a read transaction.
 func (s *MemoryStore) View(cb func(ReadTx)) {
+	defer metrics.StartTimer(viewLatencyTimner)()
```

Comment:
I'm not sure this is a useful metric. `Txn` and `Commit` are lockless and should be nearly instantaneous. The time taken is going to be dominated by `cb`, which could be doing many different things depending on the context. Without knowing which callback is invoked, statistics on the duration of `View` won't really tell us much. It may be more useful to instrument individual call sites.

But I suppose having this metric doesn't hurt, either.

Reply:
Agreed.